### PR TITLE
To Do list: list structure

### PR DESCRIPTION
**Pull request by Paul-js-hub**
### In this project I:

- Set up a project with webpack
- Created an index.js file and set an array of some simple to do tasks (array of objects).
- Wrote a function to iterate over the tasks array and populate an HTML list item element for each task.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello World
```
Comment (brytebee) [inline README.md]: - [ ] **Required:** To have more professional documentation, double-check every part it. Kindly update the title to the current.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+## Author
+👤 **Paul Onchera**
+
+- GitHub: [@githubhandle](https://github.com/Paul-js-hub/)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/paul-onchera-499a57144/)
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (brytebee) [inline README.md:51]: - [ ] **Optional:** Your license leads to a 404. Kindly update it with [the template.](https://github.com/microverseinc/readme-template) Ensure to edit to match your purpose.

File: src/modules/get-todos.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    description: 'Read WebPack Tutorial',
+    completed: true,
+    index: 1,
+  },
+  {
+    description: 'Do Laundry',
+    completed: true,
+    index: 2,
+  },
+];
```
Comment (brytebee) [inline src/modules/get-todos.js]: - [ ] **Required:** You have rendered the list excellently to the DOM, however, you are required to render them in the order of their index.

**_Hint:_** Consider `Array.sort((a, b) => a.index - b.index`

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @Paul-js-hub👋,

Good job so far coming so far with the progress you've made! 💪💪

## Highlights
You have done amazing in the following:
- Webpack was used to set up the project.✔️
- It has an array of simple objects containing the required keys and data types.✔️
- It renders the list to the page excellently.✔️
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AnselemOdims — APPROVED** [review]
### Hi  @Paul-js-hub 🙋‍♀️, 
Good job implementing all the project requirements for this milestone/project.

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**